packages/flutter_tools/lib/src/ios/devices.dart

```diff
@@ -173,15 +169,15 @@ class IOSDevice extends Device {
     return devices;
   }
 
-  static String _checkForCommand(
-    String command, [
-    String macInstructions = _kIdeviceinstallerInstructions,
-  ]) {
+  static String _checkForCommand(String command) {
     try {
-      command = runCheckedSync(<String>['which', command]).trim();
+      command = runCheckedSync(
```

Comment:
Is this check still needed for `ideviceinstaller` since we know that it should be in the artifact cache? If this check is not needed, then there's no need to clobber `PATH` when calling `which`, which I wouldn't be surprised to see confuse it.

Reply:
Good point.

Reply:
Done